src/OSPSuite.Core/Domain/Services/ModelConstructor.cs

```diff
@@ -148,16 +148,22 @@ private void removeUndefinedLocalMoleculeParametersIn(IModel model)
 
       private ValidationResult validateModel(ModelConfiguration modelConfiguration)
       {
-         var (_, simulationConfiguration) = modelConfiguration;
          if (!modelConfiguration.ShouldValidate)
             return new ValidationResult();
 
+         var (_, simulationConfiguration) = modelConfiguration;
+
+         //Validation needs to happen at the very end of the process
+         var reactionAndTransportValidation = validate<ValidatorForReactionsAndTransports>(modelConfiguration);
```

Comment:
@Yuri05 I need to move this at a later stage since some parameters are only created int he last step. So validation of structure cannot be performed at each step as we had it before